cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -248,7 +286,7 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
   @Test def drop_index_on_a_label_single_property() {
     prepareAndTestQuery(
       title = "Drop a single-property index",
-      text = "An index on all nodes that have a label and single property combination can be dropped with `DROP INDEX ON :Label(property)`.",
+      text = "An index on all nodes with a label and single property combination can be dropped with `DROP INDEX ON :Label(property)`.",
```

Comment:
should probably specify that this is only valid for btree indexes

Reply:
might be more updates around this that we want, might have done some of it as part of the range pr but might be good to see if any of that should be backported to this one

A bit dependent on the result of discussion with PM we have tomorrow around what we want to document